### DISCUSSION TITLE
Selectbox in streamlit python

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#734] [Q&A] [answered]
Bagaimana cara mengatur ukuran panjang selectbox di streamlit python ? 

<img width="634" alt="image" src="https://github.com/teamalgoritma/community/assets/106010284/a75c094a-02f2-4d39-963d-4dc2f4ad7e18">

Saya ingin panjangnya seperti yang di kotak kuning.

Mohon bantuannya kak, terima kasih 🙏

**Comment by divakartika** [comment, marked as answer]
Dear @yohanespm77 

Panjang selectbox sendiri tidak dapat diatur, namun dimensi widget dapat dirapikan dengan membuat kolom `st.columns()` sesuai Example pada dokumentasi berikut: https://docs.streamlit.io/library/api-reference/widgets/st.selectbox. Semoga membantu :) 

Bila informasi ini dapat menjawab pertanyaan Anda, silakan “*Mark as answer*” jawaban ini. Terima kasih!
